beaker-puppet.gemspec

```diff
@@ -34,7 +34,7 @@ Gem::Specification.new do |s|
 
   # Run time dependencies
   s.add_runtime_dependency 'beaker', '~> 4.1'
-  s.add_runtime_dependency 'in-parallel', '~> 0.1'
+  s.add_runtime_dependency 'in-parallel', '>= 0.1', '< 2.0'
```

Comment:
This shouldn't be a runtime dependency, since it's only overridden in the tests. Really, beaker itself only relies on this: https://github.com/voxpupuli/beaker-puppet/pull/214